UnitTests/Tests/FISAppDelegateSpec.m

```diff
@@ -0,0 +1,204 @@
+        it(@"returns YES that Labor Day is in Summer", ^{
+            expect([appDelegate holiday:@"Labor Day"
+                             isInSeason:@"Summer"]).to.beTruthy();
+        });
+        
+        it(@"returns NO that Christmas Day is in Summer", ^{
+            expect([appDelegate holiday:@"Christmas Day"
+                             isInSeason:@"Summer"]).to.beFalsy();
+        });
+    });
+    
+    describe(@"4 supply:isInHoliday:inSeason:", ^{
+        it(@"returns YES that barbeque is in Veteran's Day in Fall", ^{
+            expect([appDelegate supply:@"barbeque"
+                           isInHoliday:@"Veteran's Day"
+                              inSeason:@"Fall"]).to.beTruthy();
+        });
+        
+        it(@"returns YES that sunscreen is in Independence Day in Summer", ^{
+            expect([appDelegate supply:@"sunscreen"
+                           isInHoliday:@"Independence Day"
+                              inSeason:@"Summer"]).to.beTruthy();
+        });
+        
+        it(@"returns NO that sunscreen is not in New Year's Day in Winter", ^{
+            expect([appDelegate supply:@"sunscreen"
+                           isInHoliday:@"New Year's Day"
+                              inSeason:@"Winter"]).to.beFalsy();
+        });
+        
+    });
+    
+    
+    describe(@"5 addHoliday:toSeason:", ^{
+        it (@"returns the database with President's Day added as a key to Winter", ^{
+            NSString *presidentsDay = @"President's Day";
+            NSString *winter = @"Winter";
+            
+            [appDelegate addHoliday:presidentsDay
+                           toSeason:winter         ];
+            
+            expect([appDelegate.database[winter] allKeys]).to.contain(presidentsDay);
+        });
+        
+        it (@"returns the database with President's Day added as a key to Winter that points to the value of an empty NSMutableArray", ^{
+            NSString *presidentsDay = @"President's Day";
+            NSString *winter = @"Winter";
+            
+            [appDelegate addHoliday:presidentsDay
+                           toSeason:winter         ];
+            
+            expect(appDelegate.database[winter][presidentsDay]).to.haveCountOf(0);
+        });
+        
+        it (@"returns the database with President's Day not added as a key to another season", ^{
+            NSString *presidentsDay = @"President's Day";
+            NSString *winter = @"Winter";
+            
+            [appDelegate addHoliday:presidentsDay
+                           toSeason:winter         ];
+            
+            expect([appDelegate.database[@"Spring"] allKeys]).toNot.contain(presidentsDay);
+        });
+        
+    });
+    
+    describe(@"6 addSupply:toHoliday:inSeason:", ^{
+        it (@"returns the database with sunscreen added to Veteran's Day in Fall", ^{
+            NSString *sunscreen = @"sunscreen";
+            NSString *veteransDay = @"Veteran's Day";
```

Comment:
it was something i started on FISCardStreams tests to defend against typos whenever I used the same string literal in more than one place. I think `winter` just carried over from altering a copy of the previous test.